README.md

```diff
@@ -326,6 +326,15 @@ models:
           upper_bound_column: ended_at
           partition_by: customer_id
           gaps: required
+  
+  # test that each customer can have subscriptions that start and end on the same date
+  - name: subscriptions
+    tests:
+      - dbt_utils.mutually_exclusive_ranges:
+          lower_bound_column: started_at
+          upper_bound_column: ended_at
+          partition_by: customer_id
+          zero_length: allowed
```

Comment:
What do you think of this instead?
```suggestion
          zero_length_range_allowed: true
```

Reply:
I was thinking about that, but I thought it would be best to stick with the `required/allowed/not_allowed` convention used in `gaps` parameter. Happy to change if you feel strongly.

Reply:
I think a true/false is better here — had to use an enum for the above since there were three different options 😓

Reply:
All set :)

One other thing:
I feel like the lack of negative test coverage is concerning for this project. I would like to be able to make test cases where the test (specifically, the test that is meant to be reused in other code) fails. As it stands we only have tests where the tested model passes the test.

One kluge that we could use to solve this quickly is to put tags on tests that should fail, exclude them from the standard test run, and run the tests that should fail with the expectation that it will return a failure code.

To me a better way would be to add an optional flag `expect: pass/warn/fail` to the test.config and then invert the results appropriately here:
https://github.com/fishtown-analytics/dbt/blob/dev/kiyoshi-kuromiya/core/dbt/task/test.py#L95

I would love to hear your thoughts, so I can decide where to make my PR, Thanks!